sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/PythonExternalTransform.java

```diff
@@ -435,6 +437,15 @@ public OutputT expand(InputT input) {
     }
   }
 
+  public List<External.ExpansionInfo> getExpansionInfoList() {
+    return ImmutableList.of(
+        External.ExpansionInfo.create(
+            "beam:transforms:python:fully_qualified_named",
+            generatePayload().toByteArray(),
+            expansionService,
+            External.getFreshNamespaceIndex()));
```

Comment:
This has side effects, shouldn't be called multiple times (especially from a getter) as it returns a different value each time.